src/App.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import "./App.css";
+import SampleMarkup from "./SampleMarkup";
+
+import AceEditor from "react-ace";
+
+// import "ace-builds/src-noconflict/theme-github";
+// import "ace-builds/src-noconflict/mode-css";
+
+// import "ace-builds/webpack-resolver";
+require("ace-builds/src-noconflict/mode-css");
+require("ace-builds/src-noconflict/theme-github");
```

Comment:
Why the sudden use of require here? Did the import fail? Doesn't seem necessary to switch.

Reply:
Syntax highlighting doesn't work for some reason - was trying things to fix that. I'll look at your app since you're using AceEditor and it works in yours! 🥇